patches/chrome-browser-resources-settings-people_page-import_data_dialog.html.patch

```diff
@@ -16,6 +16,12 @@ index fdd6f9d2265fe069d159ceed6e1e7ec561a2915e..e0dee6f2af7b92dd2b7f88fefbfda92f
 +              pref="{{prefs.import_dialog_stats}}"
 +              label="$i18n{importStats}">
 +          </settings-checkbox>
++          <settings-checkbox
```

Comment:
I think there might be a way to do a template `include` here that could wrap all of this and move the actual source to brave-core. Not 100% sure though

Reply:
in either case for this and the one below, do we need separate checkboxes for all of these? If we do that's fine, just seems like it could be consolidated. Also won't this show up for non-Brave imports? That might be a little confusing?

Reply:
> in either case for this and the one below, do we need separate checkboxes for all of these?

I think we do want separate checkboxes, again so users can decide what specific types of data they want to import from Brave. e.g. I can imagine plenty of users might want to keep their stats but their wallet hasn't been working and they'd like to start over fresh there.

> Also won't this show up for non-Brave imports?

No, see how services_supported is set for each importer in `importer_list.cc`.